### PR TITLE
Add theme aware file system loader

### PR DESCRIPTION
**Pull request by wizhippo**

### CODE REVIEW

File: DependencyInjection/Compiler/ThemeCompilerPass.php [hunk 1]
```diff
@@ -24,6 +24,10 @@ public function process(ContainerBuilder $container)
 
         $container->setAlias('templating.cache_warmer.template_paths', 'liip_theme.templating.cache_warmer.template_paths');
 
+        $twigFilesystemLoaderDefinition = $container->getDefinition('twig.loader.filesystem');
+        $twigFilesystemLoaderDefinition->setClass($container->getParameter('liip_theme.filesystem_loader.class');
```
Comment (lsmith77) [inline DependencyInjection/Compiler/ThemeCompilerPass.php]: missing a closing parenthesis:
https://travis-ci.org/liip/LiipThemeBundle/jobs/59416071#L256

**Comment by wizhippo** [comment]
Looks like symfony/twig-bundle is needed as a dependency now?

**Comment by lsmith77** [comment]
you need to just mock the service in https://github.com/liip/LiipThemeBundle/blob/master/Tests/DependencyInjection/Compiler/ThemeCompilerPassTest.php

**Comment by wizhippo** [comment]
Please take a quick look at the modified test for any omissions.  I haven't had the requirement to have to write many tests.

File: Tests/DependencyInjection/Compiler/ThemeCompilerPassTest.php [hunk 2]
```diff
@@ -20,10 +20,21 @@ public function testProcess()
             ->method('setAlias')
         ;
 
+        $containerMock->expects($this->exactly(2))
+                      ->method('getParameter')
```
Comment (lsmith77) [inline Tests/DependencyInjection/Compiler/ThemeCompilerPassTest.php]: can you change the identing to match the rest of the code?

**Comment by lsmith77** [comment]
looks good .. aside from the minor detail

**Pull request merged**